src/App.tsx

```diff
@@ -1,61 +1,137 @@
+import { Todo } from './interfaces/Todo';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+
+const initialTodos: Todo[] = todosFromServer.map(todo => {
+  const user = usersFromServer.find(client => client.id === todo.userId)!;
```

Comment:
[CODE KNOWLEDGE] - You are using a non-mutating array method `find` here, so you don't need to create a copy of the array.